x/ccv/child/keeper/keeper.go

```diff
@@ -22,36 +22,52 @@ import (
 
 // Keeper defines the Cross-Chain Validation Child Keeper
 type Keeper struct {
-	storeKey         sdk.StoreKey
-	cdc              codec.BinaryCodec
-	paramSpace       paramtypes.Subspace
-	scopedKeeper     capabilitykeeper.ScopedKeeper
-	channelKeeper    ccv.ChannelKeeper
-	portKeeper       ccv.PortKeeper
-	connectionKeeper ccv.ConnectionKeeper
-	clientKeeper     ccv.ClientKeeper
+	storeKey          sdk.StoreKey
+	cdc               codec.BinaryCodec
+	paramSpace        paramtypes.Subspace
+	scopedKeeper      capabilitykeeper.ScopedKeeper
+	channelKeeper     ccv.ChannelKeeper
+	portKeeper        ccv.PortKeeper
+	connectionKeeper  ccv.ConnectionKeeper
+	clientKeeper      ccv.ClientKeeper
```

Comment:
You don't really need this since they're all accessible by `ibcCoreKeeper`

Reply:
The clients are? makes sense, however that logic is actually outside of the scope of this PR (I only modified further lines below - just weird github highlighting). I'll make a PR removing the client keeper in a separate PR in the future (or feel free to do the same)